Source/APIClient/Request.swift

```diff
@@ -19,6 +19,7 @@ public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
     static var authToken: String? { get }
+    static var authTokenHeader: String? { get }
```

Comment:
When designing protocols I generally follow the rule that its member variables/functions should be driven by the objects that reference that protocol type directly (i.e. a class that references a variable of type `Request` as opposed to a specific class that conforms to `Request`). The variables/functions exposed by the protocol should be kept to a minimum, and represent only what that referencing class needs in order to operate properly.